src/mono/wasm/testassets/WasmBasicTestApp/App/wwwroot/main.js

```diff
@@ -218,10 +222,12 @@ try {
                 await INTERNAL.loadLazyAssembly(`LazyLibrary${lazyAssemblyExtension}`);
                 const { LazyLibrary } = await getAssemblyExports("LazyLibrary");
                 const resLazy = LazyLibrary.Foo.Bar();
+                await delay(200);
```

Comment:
Do you have an actual time dependency here or are you just trying to yield to let something else run. Maybe you can use `Promise.resolve()` if it's the latter